techmatters/flex-plugins
# Style the 'single resource view'

## Description
This PR adds [the styles](https://marvelapp.com/prototype/ieh6cb7/screen/88434650) to the "view single resource" screen.

![Screenshot from 2023-01-27 17-46-51](https://user-images.githubusercontent.com/15805319/215193922-a5c3e47d-ec93-4d2a-a07a-dd467daf2cdc.png)

NOTES: 
- ~The last commit of this PR breaks the resources view, since the data does not matches the shape expected.~  The last commit does not shows any attributes since the returned example resource does not contains any. You must hardcode a mocked response (like in https://github.com/techmatters/flex-plugins/pull/1207/commits/07cccd6dbb76ffbbaaf8deca680749cb5b529975).
- This PR just styles according to the mocks, assuming the resources as described in that particular scenario, for KHP. I assume we'll have some sort of layout specification per account, that tells this view what it should display for each resource. (Already talked a bit about this with Steve, but is something still unclear to me).

### Checklist
- [x] [Corresponding issue has been opened](https://tech-matters.atlassian.net/browse/CHI-1419)
- [n/a] New tests added
- [n/a] Feature flags added
- [n/a] Tested for chat contacts
- [n/a] Tested for call contacts

### Verification steps
- `❯ git checkout 07cccd6 -- src/services/ResourceService.ts` so the API returns a mocked resource.
- Confirm that the view resource screen matches the styles.
